Website/Install/Config/09.04.00.config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+  <nodes configfile="web.config">
+    <node path="/configuration/system.web/compilation" action="update" key="targetFramework" value="4.7.2" collision="ignore" />
```

Comment:
```suggestion
    <node path="/configuration/system.web/compilation" action="updateattribute" name="targetFramework" value="4.7.2" />
```

Reply:
See comment above, thanks for the feedback and update is on it's way